### PR TITLE
test(protocol): integration tests for isMessageFailed

### PR DESCRIPTION
**Pull request by RogerLamTd**

### CODE REVIEW

**Comment by cyberhorsey** [comment]
@RogerLamTd any update on this?

**Comment by RogerLamTd** [comment]
> @RogerLamTd any update on this?

Working with David on some issues, managed to start the geth node instead but since they're both --dev nodes they are defaulted to chainId 1337 which causes issues; may want to add a check for duplicate chain IDs in the future as well if it's an edge case. Trying to figure out a way to start one with a custom chainId, and a couple other issues

**Comment by davidtaikocha** [comment]
> > @RogerLamTd any update on this?
> 
> Working with David on some issues, managed to start the geth node instead but since they're both --dev nodes they are defaulted to chainId 1337 which causes issues; may want to add a check for duplicate chain IDs in the future as well if it's an edge case. Trying to figure out a way to start one with a custom chainId, and a couple other issues

pushed a new commit to fix your issues, plz try again.

**Comment by RogerLamTd** [comment]
Reverted bridge changes mean all bridge tests work fine now, but TaikoL1 tests are having trouble on the l2 geth node changes. Still investigating but would appreciate any help!

File: packages/protocol/test/L1/TaikoL1.integration.test.ts [hunk 1]
```diff
@@ -55,8 +56,10 @@ describe("integration:TaikoL1", function () {
             0,
             proposerSigner
         );
+        console.log("proposer deployed");
 
         prover = new Prover(taikoL1, l2Provider, proverSigner);
+        console.log("proposer deployed");
```
Comment (cyberhorsey) [inline packages/protocol/test/L1/TaikoL1.integration.test.ts:62]: this should be "prover", but do we also want to remove these logs?

File: packages/protocol/test/L1/TaikoL1.integration.test.ts [hunk 2]
```diff
@@ -120,14 +123,17 @@ describe("integration:TaikoL1", function () {
             );
             expect(proposedEvent).not.to.be.undefined;
             expect(proposedEvent.args.meta.commitSlot).to.be.eq(commitSlot);
+            console.log("passed");
 
             const proposedBlock = await taikoL1.getProposedBlock(
                 proposedEvent.args.meta.id
             );
+            console.log("passed2");
```
Comment (cyberhorsey) [inline packages/protocol/test/L1/TaikoL1.integration.test.ts:131]: we should remove all of these

**Review by cyberhorsey — CHANGES_REQUESTED** [review]

File: packages/protocol/test/test_integration.sh [hunk 3]
```diff
@@ -63,21 +71,13 @@ function waitTestNode {
 waitTestNode http://localhost:18545
 waitTestNode http://localhost:28545
 
-# Import ETHs from the random pre-allocated developer account to the test account
-docker run -d \
-  --name $TEST_IMPORT_TEST_ACCOUNT_ETH_JOB_NAME \
```
Comment (cyberhorsey) [inline packages/protocol/test/test_integration.sh:68]: dont we still need this if test type is not integration?

**Review by cyberhorsey — DISMISSED** [review]

**Comment by RogerLamTd** [comment]
@cyberhorsey I separated the bridge integration tests from the rest of the integration tests, but the build run doesn't include them; I'm not too clear on how to edit that, could you take a look? David believes the occasional integration test failures are from the switching to 2 geth nodes, so this change should prevent that issue.

**Review by davidtaikocha — APPROVED** [review]

**Review by cyberhorsey — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Review by cyberhorsey — APPROVED** [review]

**Pull request merged**